webapp/src/main/webapp/iaf/frankdoc/js/controllers.js

```diff
@@ -37,6 +37,12 @@ angular.module('iaf.frankdoc').controller("main", ['$scope', '$http', 'propertie
 			}
 			for(let i in elements) {
 				let element = elements[i];
+				if(element.attributes) {
+					element.showAttributes = true;
```

Comment:
Niels commented that the same can be done with ng-if and a || operator. I will eliminate showAttributes.

Reply:
Done